File: JavaScript_Courses/JavaScript_Basics/Preparation/05_While_Loop_Lab/06_Max_Number.js
function maxNumber(input) {
    let currentMax = Number(input[0]);

    let i = 1;
    while (true) {
        let currentCommand = input[i];
        i++;

        if (currentCommand == "Stop") {
            break;
        }

        if (Number(currentCommand) > currentMax) {
            currentMax = Number(currentCommand);
        }
    }

    console.log(currentMax);
}


// maxNumber(["100",
//     "99",
//     "80",
//     "70",
//     "Stop"]);

File: Python_Courses/Python_Basics/Python_PB_2024_06/05_While_Loop_Lab/06_Max_Number.py
# която до получаване на командата "Stop", чете цели числа
import sys

max_number = -sys.maxsize

command = input()  # "Stop" or "13"
while command != "Stop":
    current_number = int(command)  # int('13') == 13
    if current_number > max_number:
        max_number = current_number

    command = input()

print(max_number)
